tests/export/html/test_field_code.py

```diff
@@ -7,11 +7,58 @@
 )
 
 
-from pydocx.openxml.packaging import MainDocumentPart
+from pydocx.openxml.packaging import MainDocumentPart, StyleDefinitionsPart
 from pydocx.test import DocumentGeneratorTestCase
 from pydocx.test.utils import WordprocessingDocumentFactory
 
 
+class HeadingTestCase(DocumentGeneratorTestCase):
```

Comment:
You have a testcase named `HyperlinkFieldCodeTestCase` in this module, so wouldn't calling this `HeadingFieldCodeTestCase` be more consistent?